scheduler/tests.py

```diff
@@ -537,6 +537,8 @@ class TestSchedulableJob(TestBaseJob):
         # Currently ScheduledJob and RepeatableJob
         JobClass = BaseJob
         JobClassFactory = BaseJobFactory
+        RepeatableJobClassFactory = RepeatableJobFactory
+        ScheduledJobClassFactory = ScheduledJobFactory
```

Comment:
Keep JobClassFactory, but separate the tests that are to be different from Repeatable (with and without Repeat=None) and Scheduled (repeat does not exist).

Reply:
Why not add your test to `TestRepeatableJob` instead?